src/image/PhotoCalib.cc

```diff
@@ -95,7 +95,7 @@ void toNanojanskyVariance(ndarray::Array<float const, 2, 1> const &instFlux,
 }
 
 double toMagnitudeErr(double instFlux, double instFluxErr, double scale, double scaleErr) {
-    return 2.5 / log(10.0) * hypot(instFluxErr / instFlux, scaleErr / scale);
+    return 2.5 / std::log(10.0) * hypot(instFluxErr / instFlux, scaleErr / scale);
```

Comment:
This was necessary to use the natural logarithm and not a logging function, and whether or not we decide to log, we should perhaps keep this change.